tasks/variables/variables.js

```diff
@@ -7,42 +7,52 @@
  * TODO create variable named int and set value equal to 1
  */
 
-
+var int = 1;
```

Comment:
`var` is deprecated keyword for variables. Please use `let` or `const`.

Read more: https://github.com/airbnb/javascript#references

Please check whole file.